termin.py

```diff
@@ -200,7 +200,7 @@ def get_available_appointment_types():
             'Fachrichtungswechsel',
             'Facharztausbildung',
             'Niederlassungserlaubnis allgemein',
-            'Niederlassungserlaubnis Blaue Karte EU',
+            'Niederlassungserlaubnis Blaue Karte EU - Beratung/ Antragstellung',
```

Comment:
Wasn't this fixed in previous PR? Have you rebased `git fetch <upstream> && git rebase <upstream>/master`?

Reply:
I think I forgot to do it before creating the other branch.

Reply:
Could you please rebase then? `git fetch <upstream> && git rebase <upstream>/master`